andela/devrepublic-bn-backend
# #170947569 create accommodation facilities and rooms

#### What does this PR do?
- Travel admin is able to create a facility feature
- Travel admin is able to create a room in the facility

#### Description of Task to be completed?
- setup table with sequelize
- create routes to the controller
- create controller for adding facilities and rooms
- write tests for the controllers

#### How should this be manually tested?
- clone the repo
- run `npm install` to install all dependencies
- make sure you have postgres installed
- run `npm run create` to create tables
- run `npm run seed:dev` to seed users in the db
- go to postman, using POST method, log in on this address `https:\\{BASE_URL}/api/v1/auth/login`
- login with this account email:`jdev@andela.com` and password `Bien@BAR789`
- use the user's token to access these routes `localhost:8000/api/v1/user/create/facility` & `localhost:8000/api/v1/user/create/facility`
- In the first route, enter JSON object including facilityName & location
- In the second route, enter JSON object including the faclityId, roomName & type.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#170947569](https://www.pivotaltracker.com/story/show/170947569)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/30793016/74931595-9aa14600-53e8-11ea-9035-9f1d523e24ee.png)
![image](https://user-images.githubusercontent.com/30793016/74931607-a0972700-53e8-11ea-8ec3-428b6485ad0c.png)
![image](https://user-images.githubusercontent.com/30793016/74931614-a55bdb00-53e8-11ea-9c4b-6fff3113b381.png)
